source/dub/init.d

```diff
@@ -158,7 +158,20 @@ private void initDeimosPackage(Path root_path, ref PackageRecipe p, scope void d
 private void writeGitignore(Path root_path)
 {
 	write((root_path ~ ".gitignore").toNativeString(),
-		".dub\ndocs.json\n__dummy.html\n*.o\n*.obj\n__test__*__\n");
+q"{.dub
+docs.json
+__dummy.html
+docs/
+*.so
+*.dylib
+*.dll
+*.a
+*.lib
+*.exe
+*.o
+*.obj
+__test__*__
```

Comment:
The test pattern has changed to "(project name)-test-(configuration)[.exe]" recently.